vue.config.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
 	css: {
 		extract: false,
 	},
+	configureWebpack: (config) => {
+		config.externals = [/^@?primevue\/.*$/];
```

Comment:
Now I see `design-system.umd.js` has the Prime code removed and this added:
```
require("vue"), require("primevue/skeleton"), require("primevue/config"), require("@primevue/themes/aura"), require("primevue/tooltip")
```
where previously it was just `require("vue")`. I'm still not sure how it works because I see no `primevue/skeleton` in the final platform code. I can live with it for now, as it uses the same method as Vue.

Reply:
https://github.com/bethinkpl/wnl-platform/blob/dc7b33c338be81aee248238fc5d7bcb05e2f893f/vite.config.ts#L78-L79
Gotcha 👍